fuji_server/evaluators/fair_evaluator_license.py

```diff
@@ -29,14 +31,26 @@ class FAIREvaluatorLicense(FAIREvaluator):
 
     def __init__(self, fuji_instance):
         FAIREvaluator.__init__(self, fuji_instance)
-        self.set_metric("FsF-R1.1-01M")
+        self.set_metric(["FsF-R1.1-01M", "FRSM-15-R1.1"])
 
         self.output = []
         self.license_info = []
 
+        # Create map from metric test names to class functions. This is necessary as functions may be reused for different metrics relating to licenses.
+        self.metric_test_map = {  # overall map
+            "testLicenseIsValidAndSPDXRegistered": ["FsF-R1.1-01M-2", "FRSM-15-R1.1-3"],
+            "testLicenseMetadataElementAvailable": ["FsF-R1.1-01M-1", "FRSM-15-R1.1-1"],
+            "testLicenseFileAtRoot": ["FRSM-15-R1.1-CESSDA-1"],
+            "testLicenseInHeaders": ["FRSM-15-R1.1-CESSDA-2"],
+            "testLicenseForBundled": ["FRSM-15-R1.1-2"],
+            "testBuildScriptChecksLicenseHeader": ["FRSM-15-R1.1-CESSDA-3"],
+        }
+
     def setLicenseDataAndOutput(self):
         self.license_info = []
         specified_licenses = self.fuji.metadata_merged.get("license")
+        if specified_licenses is None:  # try GitHub data
```

Comment:
I think here it has to be tested first if a FRSM metric is used

Reply:
Good call, I'll add that. Just a note, I've also been thinking about merging the `github_data` dictionary into the `merged_metadata` but had to focus on other aspects first - might want to consider this further down the line though.

Reply:
Added in https://github.com/pangaea-data-publisher/fuji/pull/478/commits/732b48e80007c99ebc43e1767cf7900584823062.